LDAR_Sim/src/ldar_sim.py

```diff
@@ -188,4 +188,4 @@ def save_results(self, data: pd.DataFrame, filename: str) -> None:
         if data is None:
             return
         filepath: Path = self._output_dir / filename
-        data.to_csv(filepath, index=False)
+        data.to_csv(filepath, index=False, float_format="%.5f")
```

Comment:
Clean Code: This might be unnecessary, as I think this is the default if float_format is not provided. Could you check before merging this?

Reply:
From what I could find default behavior for pandas is to show things up to 6 decimal places, and in scientific notation.

Reply:
Pandas version 2.20 documentation shows no default float format so I think specifying this is fine